src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75d;
+    private int threshold;
+    private int capacity;
+    private int size;
+    private Node<K, V>[] buckets;
+
+    public static class Node<K, V> {
+        private int hash;
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        buckets = new Node[DEFAULT_CAPACITY];
+        size = 0;
+        capacity = DEFAULT_CAPACITY;
```

Comment:
```suggestion
        buckets = new Node[DEFAULT_CAPACITY];
        capacity = DEFAULT_CAPACITY;
```

we don't neet to initialize size by 0, it will be initialized 0 by default

Reply:
I need to reset it, because when the array is overwritten with a new one, the put method is called, and the size is recalculated there

Reply:
You set size to 0 twice? 
<img width="519" alt="image" src="https://user-images.githubusercontent.com/68428976/161698636-ad0525f6-d52a-4fdf-8698-d704cc7a5db1.png">

Reply:
The first time I set the values ​​to 0 in the hashmap constructor
In this method, it is needed because the put method is called in the transfer method, and the size increment  in the put method.
If we remove =0 from here, then when resizing the size value will be twice the actual size